ocaml/xapi/xapi_vgpu_type.ml

```diff
@@ -288,11 +290,14 @@ module Nvidia_old = struct
                   (List.assoc "plugin0.num_heads" args) in
               let max_instance = Int64.of_string
                   (List.assoc "plugin0.max_instance" args) in
+              (* Define "0" here because now we are using new way to read from nvidia conf file  *)
+              let type_id = "0" in
```

Comment:
Should this not be an integer that is converted to a string representation when needed?

Reply:
Here is the old way of getting the config from Nvidia. I think now they are deprecated. I just make the compile pass and leave a comment above. We could delete the old way of this actually :)

Reply:
Let's delete the old code (in a separate commit), as it really won't work anymore now that we need this `type_id`.

Reply:
OK, I will delete in a seperate commit.